mediawiki/install_extensions.sh

```diff
@@ -12,6 +12,7 @@ declare -a extensions=( \
     CommonsMetadata-REL1_28-e3c0bbe \
     MultimediaViewer-REL1_28-b426dc3 \
     SandboxLink-REL1_28-aa109b7 \
+    SpamBlacklist-REL1_28-5940064 \
```

Comment:
As @DefCon-007 pointed out, this extension is bundled with mediawiki, so we don't need to download it again.